### PR TITLE
v1.5.2 release

### PR DESCRIPTION
**Pull request by grst**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/demultiplex/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/demultiplex _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 1.5.2dev (UNRELEASED)
```
Comment (christopher-mohr) [inline CHANGELOG.md:6]:
```suggestion
## 1.5.2- 2024-10-07
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### `Changed`
+
+- [#253](https://github.com/nf-core/demultiplex/pull/256) Updated all modules and subworkflows, and added new bclconvert test profile.
```
Comment (christopher-mohr) [inline CHANGELOG.md:10]:
```suggestion
- [#256](https://github.com/nf-core/demultiplex/pull/256) Updated all modules and subworkflows, and added new bclconvert test profile.
```

Comment (christopher-mohr) [reply CHANGELOG.md:10]: Is this the right PR?

File: tests/pipeline/test_pe.nf.test.snap [hunk 3]
```diff
@@ -76,9 +76,9 @@
...
         "content": [
-            "bcl2fastq_lane_counts.txt:md5,617ee87ca15a85232d9fd58808453204",
-            "fastp_filtered_reads_plot.txt:md5,bc473fbbdae16865669a66d5c3180ca3",
-            "bcl2fastq_sample_counts.txt:md5,6454d154d69ffa563878f5b7fd3d9879"
+            "bcl2fastq_lane_counts.txt:md5,d6054e1ec3c2e8d33c3fdee5b250dfbc",
+            "fastp_filtered_reads_plot.txt:md5,136a1d755ba6e790227f80cfcb4c1d62",
+            "bcl2fastq_sample_counts.txt:md5,93cb0cd1aee1b464f0e640e1c3d0c142"
         ],
         "meta": {
             "nf-test": "0.9.0",
```
Comment (christopher-mohr) [inline tests/pipeline/test_pe.nf.test.snap:84]: Is `0.9.0` ok here?

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
Couple of minor suggestions/questions. Apart from that, LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (grst) [reply CHANGELOG.md:6]: waiittt, isn't this updated by `nf-core bump version`?

good catch :)

File: CHANGELOG.md (hunk 2 above)
Comment (grst) [reply CHANGELOG.md:10]: should be #263 I think

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,11 +5,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
 ## 1.5.1
 
 ### `Fixed`
 
 - [#253](https://github.com/nf-core/demultiplex/pull/253) Fixed mkfastq output channels.
+- [#264](https://github.com/nf-core/demultiplex/pull/264) Fixed adapter removal for input samplesheets without lane information.
```
Comment (grst) [inline CHANGELOG.md:17]:
```suggestion
- [#265](https://github.com/nf-core/demultiplex/pull/265) Fixed adapter removal for input samplesheets without lane information.
```

File: tests/pipeline/test_pe.nf.test.snap (hunk 3 above)
Comment (grst) [reply tests/pipeline/test_pe.nf.test.snap:84]: It's the latest version... is there a particular reason you are asking?

Comment (christopher-mohr) [reply tests/pipeline/test_pe.nf.test.snap:84]: Because it was changed to `0.8.4` for others tests it seems.

**Review by christopher-mohr — APPROVED** [review]
LGTM!

**Pull request merged**